README.md

```diff
@@ -43,25 +43,39 @@ For an example of using ALB with ECS look no further than the [hashicorp example
 * `alb_dns_name` - DNS CNAME of the ALB created.
 * `alb_zone_id` - Route53 `zone_id` of the newly minted ALB.
 * `target_group_arn` - `arn` of the target group. Useful for passing to your Auto Scaling group module.
+* `root_principle_id` - the id of the AWS root user within this region. See [docs here]('http://docs.aws.amazon.com/elasticloadbalancing/latest/classic/enable-access-logs.html#attach-bucket-policy').
 
 ## Usage example:
-A full example set is contained in the [example directory](example/). Here's the gist:
-1. Set the input variables from above in [variables.tf](example/variables.tf).
-2. Define the ALB module using the following in your [main.tf](example/main.tf):
+A full example set is contained in the [test/fixtures directory](test/fixtures). Here's the gist:
+1. Set the input variables from above in [variables.tf](test/fixtures/variables.tf).
+2. Define the ALB module using the following in your [main.tf](test/fixtures/main.tf):
 ```
 module "alb" {
-  source              = "github.com/brandoconnor/tf_aws_alb"
-  alb_security_groups = "${var.security_group_id_list}"
+  source              = "github.com/terraform-community-modules/tf_aws_alb/alb"
```

Comment:
There should be double slash before directory name, like this - `github.com/terraform-community-modules/tf_aws_alb//alb`. Via https://www.terraform.io/docs/modules/sources.html